### PR TITLE
Add support for extra custom docker configuration

### PR DESCRIPTION
**Pull request by noliveleger**
Sometimes for custom install, some variables or configurations may be needed to be passed docker containers.
All custom configuration added to `*.override.yml` are lost upgrade or when setup is run.
This version lets users add another file to `kobo-docker` folder to persist any custom configurations.

### CODE REVIEW

**Comment by noliveleger** [comment]
@joshuaberetta, sorry I mixed the feature changes with some `Black formatting` refactoring. Let me know if you prefer to have formatting change reverted and moved to another PR.

File: helpers/command.py [hunk 1]
```diff
@@ -495,3 +507,58 @@ def version(cls):
...
+            custom_file = '{}/docker-compose.backend.{}.custom.yml'.format(
+                dict_['kobodocker_path'],
+                backend_server_role
+            )
+            if not os.path.exists(custom_file):
+                message = (
+                    'Please create your custom configuration in\n'
+                    '`{custom_file}`.'
+                ).format(custom_file=custom_file)
+                CLI.framed_print(message, color=CLI.COLOR_INFO, columns=75)
```
Comment (joshuaberetta) [inline helpers/command.py:557]: Maybe we should increase the number of columns so it doesn't break onto the next line?
```
╔═════════════════════════════════════════════════════════════════════════╗
║                                                                         ║
║ Please create your custom configuration in                              ║
║ `/home/josh/kobo-tmp/kobo-docker/docker-                                ║
║ compose.backend.primary.custom.yml`.                                    ║
║                                                                         ║
╚═════════════════════════════════════════════════════════════════════════╝
```
Rather
```
╔═══════════════════════════════════════════════════════════════════════════════╗
║                                                                               ║
║ Please create your custom configuration in                                    ║
║ `/home/josh/kobo-tmp/kobo-docker/docker-compose.backend.primary.custom.yml`.  ║
║                                                                               ║
╚═══════════════════════════════════════════════════════════════════════════════╝
```

File: helpers/config.py [hunk 2]
```diff
@@ -256,18 +248,9 @@ def frontend(self):
...
-    def frontend_questions(self):
-        """
-        Checks whether questions are front-end only
-
-        Returns:
-            bool
-        """
-        return not self.multi_servers or self.frontend
+        return (
+            not self.multi_servers or self.__dict['server_role'] == 'frontend'
```
[the remark this answers is not in the document]
Comment (noliveleger) [reply helpers/config.py:252]: Yes, except that inside the property itself. It cannot call itself ;-)

File: helpers/command.py [hunk 3]
```diff
@@ -495,3 +507,58 @@ def version(cls):
...
+                dict_['kobodocker_path']
+            )
+            if not os.path.exists(custom_file):
+                message = (
+                    'Please create your custom configuration in\n'
+                    '`{custom_file}`.'
+                ).format(custom_file=custom_file)
+                CLI.framed_print(message, color=CLI.COLOR_INFO, columns=75)
+                input('Press any key when it is done...')
+
```
[the remark this answers is not in the document]
Comment (noliveleger) [reply helpers/command.py:541]: Hmmm. Very good point ;-)

**Pull request merged**